Source/Demagnetization.cpp

```diff
@@ -369,6 +429,23 @@ void Demagnetization::CalculateH_demag(Array<MultiFab, AMREX_SPACEDIM>& Mfield,
     ParallelCopy(H_demagfield[0], dest_box, Hx_large, 0, 0, 1, IntVect(0), dtos);
     ParallelCopy(H_demagfield[1], dest_box, Hy_large, 0, 0, 1, IntVect(0), dtos);
     ParallelCopy(H_demagfield[2], dest_box, Hz_large, 0, 0, 1, IntVect(0), dtos);
+    
+    BoxArray ba = H_demagfield[0].boxArray();
+    DistributionMapping dm = H_demagfield[0].DistributionMap();    
+
+    MultiFab Plt_demag(ba_large, dm_large, 3, 0);
+
+    MultiFab::Copy(Plt_demag, Hx_large, 0, 0, 1, 0);
+    MultiFab::Copy(Plt_demag, Hy_large, 0, 1, 1, 0);
+    MultiFab::Copy(Plt_demag, Hz_large, 0, 2, 1, 0);
+    
+    
+    WriteSingleLevelPlotfile("H_demag", Plt_demag,
+    {"Hx",
+    "Hy",
+    "Hz"},
+    geom_large, 0., 0);
+    
```

Comment:
Delete this too, the final H_demag appears in the regular plt plot files each step.